packages/backend/src/openapi/webhooks.yaml

```diff
@@ -0,0 +1,275 @@
+  outgoingPaymentCompleted:
+    post:
+      requestBody:
+        description: Notify account servicing entity that an outgoing payment completed. Remaining funds should be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/outgoingPayment'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+  outgoingPaymentFailed:
+    post:
+      requestBody:
+        description: Notify account servicing entity that an outgoing payment failed and won't be retried. Funds should be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/outgoingPayment'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+  webMonetization:
+    post:
+      requestBody:
+        description: Notify account servicing entity that a Web Monetization payment was received and funds should be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/webMonetization'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+
+components:
+  schemas:
+    incomingPayment:
+      required:
+        - id
+        - type
+        - attempts
+        - processAt
+        - data
+      properties:
+        id:
+          type: string
+          format: uuid
+        type:
+          type: string
+          enum:
+            - incoming_payment.completed
+            - incoming_payment.expired
+        attempts:
+          type: integer
+        processAt:
+          type: string
+          format: date-time
+        data:
+          type: object
+          required:
+            - id
+            - paymentPointerId
+            - createdAt
+            - updatedAt
+            - expiresAt
+            - receivedAmount
+            - completed
+          properties:
+            id:
+              type: string
+              format: uuid
+            paymentPointerId:
+              type: string
+              format: uuid
+            completed:
+              type: boolean
+            incomingAmount:
+              $ref: ./schemas.yaml#/components/schemas/amount
+            receivedAmount:
+              $ref: ./schemas.yaml#/components/schemas/amount
+            description:
+              type: string
+            externalRef:
+              type: string
+            createdAt:
+              type: string
+              format: date-time
+            updatedAt:
+              type: string
+              format: date-time
+            expiresAt:
+              type: string
+              format: date-time
+        statusCode:
+          type: integer
+        withdrawal:
```

Comment:
Do we necessarily need `withdrawal`? it's not being used in the MAP webhooks handler, and looking at the webhook publishers, the only information that isn't already on the `data`  is the `assetId`: something that we could just include in the `data` object as well IMO

Reply:
I think it's only used by the outgoing payment event. 
@wilsonianb can you explain its use?

Reply:
I was looking at this: https://github.com/interledger/rafiki/blob/main/packages/backend/src/webhook/model.ts#L8

Is it just `id`, `type`, and `data` for all of them?

Reply:
yeah, I think the webhook service code I linked :point_up_2: is the only place we're sending webhook requests

Reply:
This one ^ just adds it to the DB to prepare for sending. It looks the `event.withdrawal` doesn't gets added to the webhook `POST` request (the one that @wilsonianb commented), since body just has `id` `type` `data`.

Maybe we can even just remote `withdrawal` from the code completely on a follow-up PR?